packages/grid/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@react-aria-components/grid",
+  "version": "0.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "author": "Juan Carlos Medina",
+  "license": "MIT",
+  "dependencies": {
+    "@react-aria-components/core": "^0.0.0"
+  }
```

Comment:
Really this thing should have a dependency (or peer dependency) on prop-types, as well.